site/app/controllers/MiscController.php

```diff
@@ -348,11 +348,7 @@ public function downloadSubmissionZip($gradeable_id, $user_id, $version, $is_ano
             $this->core->redirect($this->core->buildCourseUrl());
         }
 
-        $graded_gradeable = $this->core->getQueries()->getGradedGradeable($gradeable, $user_id, $gradeable->isTeamAssignment());
-
-        if ($gradeable->isTeamAssignment()) {
-            $graded_gradeable = $this->core->getQueries()->getGradedGradeable($gradeable, null, $user_id);
-        }
+        $graded_gradeable = $this->core->getQueries()->getGradedGradeable($gradeable, $user_id, $user_id);
```

Comment:
I'm not sure what was the intention of the if statement, but it seems like this bug is caused by the incorrect arguments order in the `getGradedGradeable()` function, the third parameter takes in a `team` object, so we can just leave it as `null`.
```suggestion
        $graded_gradeable = $this->core->getQueries()->getGradedGradeable($gradeable, $user_id, null);
```